pypose/lietensor/lietensor.py

```diff
@@ -433,8 +448,21 @@ def translation(self, input):
     def identity(cls, *size, **kwargs):
         return SE3_type.Log(SE3_type.identity(*size, **kwargs))
 
-    def randn(self, *size, sigma=1, requires_grad=False, **kwargs):
-        data = super().randn(*size, sigma=sigma, **kwargs).detach()
+    def randn(self, *size, sigma=1.0, requires_grad=False, **kwargs):
+        #  convert different types of inputs to SE3 sigma
+        if not isinstance(sigma, collections.abc.Iterable):
+            sigma = _quadruple(sigma)
+        elif len(sigma)==2:
+            rotation_sigma = _single(sigma[-1])
+            translation_sigma = _triple(sigma[0])
+            sigma = translation_sigma+rotation_sigma
```

Comment:
`sigma = translation_sigma + rotation_sigma` instead of `sigma = translation_sigma+rotation_sigma`